### PR TITLE
Add ci- prefix for gce-enormous jobs in SQ

### PR DESCRIPTION
**Pull request by krzyzacy**
https://github.com/kubernetes/test-infra/pull/1165

kubernetes/kubernetes#37156
kubernetes/kubernetes#37157

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/2085)

### CODE REVIEW

**Comment by krzyzacy** [comment]
@fejta @apelisse

**Comment by krzyzacy** [comment]
@fejta please take it from @apelisse or the baby will cry :-)

File: mungegithub/submit-queue/deployment/kubernetes/configmap.yaml [hunk 1]
```diff
@@ -80,9 +80,9 @@ data:
     ci-kubernetes-soak-gke-gci-deploy,\
     ci-kubernetes-soak-gke-gci-test,\
     ci-kubernetes-soak-gke-test,\
-    kubernetes-e2e-gce-enormous-cluster,\
-    kubernetes-e2e-gce-enormous-deploy,\
-    kubernetes-e2e-gce-enormous-teardown,\
+    ci-kubernetes-e2e-gce-enormous-cluster,\
+    ci-kubernetes-e2e-gce-enormous-deploy,\
+    ci-kubernetes-e2e-gce-enormous-teardown,\
     kubernetes-garbagecollector-gci-feature,\
```
Comment (fejta) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: This job no longer exists.

**Review by fejta — APPROVED** [review]

**Comment by apelisse** [comment]
He didn't wait for you !! 😂

**Comment by fejta** [comment]
Tell him to hold his diaper!!!

File: mungegithub/submit-queue/deployment/kubernetes/configmap.yaml (hunk 1 above)
Comment (krzyzacy) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: did you rename it to something else?

Comment (krzyzacy) [inline mungegithub/submit-queue/deployment/kubernetes/configmap.yaml]: I'll kill it in next PR

**Pull request merged**